.github/workflows/bode_ci.yaml

```diff
@@ -0,0 +1,52 @@
+name: "Bode CI"
+
+on:
+  pull_request:
+    paths:
+      - "bode/**"
+  push:
+    paths:
+      - "bode/**"
```

Comment:
I think every push means also running after pushing from local -- it may be doubled and unnecessary. Wdyt?

Reply:
Yeah, this way you can see if CI is passing without creating a PR, the feedback loop is a bit shorter. I think we can leave it like that, merge, observe if we have redundant workflow runs and then adjust accordingly if needed

Reply:
ok